src/components/SeedBank/index.tsx

```diff
@@ -112,7 +112,7 @@ export default function SeedBankDetails(): JSX.Element {
               id='timezone'
               type='text'
               value={tz.longName}
-              tooltipTitle={seedBank?.timeZone ? undefined : strings.ORG_TZ_TOOLTIP_INFO}
+              tooltipTitle={strings.TOOLTIP_TIME_ZONE_SEEDBANK}
```

Comment:
With this change, if we manually set a timezone for the seedbank, we will still see this tooltip that says it's inherited from org, and in this case it would not be inherited

Reply:
Oh, nevermind, i see it's lorem ipsum now

Reply:
Yes, I switched them all to use the placeholder and assigned the jira to Ming - to provide the values.